numba/core/untyped_passes.py

```diff
@@ -1529,7 +1531,8 @@ def run_pass(self, state):
                     fn = guard(get_definition, func_ir, value.func.name)
                     if fn is None:
                         continue
-                    if not (isinstance(fn, ir.Global) and fn.name == 'isinstance'):  # noqa: E501
+                    if not (isinstance(fn, ir.Global) and fn.name in
+                            accepted_functions):  # noqa: E501
```

Comment:
The error msg that comes after should not be referring to hardcoded `"isinstance"` anymore

Reply:
Fix and test in 1c5970d.